packages/block-library/src/latest-posts/edit.js

```diff
@@ -20,16 +16,18 @@ import {
 	Spinner,
 	ToggleControl,
 	Toolbar,
+	RadioControl,
 } from '@wordpress/components';
 import apiFetch from '@wordpress/api-fetch';
 import { addQueryArgs } from '@wordpress/url';
 import { __ } from '@wordpress/i18n';
 import { dateI18n, format, __experimentalGetSettings } from '@wordpress/date';
+import { decodeEntities } from '@wordpress/html-entities';
 import {
 	InspectorControls,
 	BlockAlignmentToolbar,
 	BlockControls,
-} from '@wordpress/block-editor';
+} from '@wordpress/editor';
```

Comment:
There are some ongoing code reorganizations and those components were moved to the new package. They will continue to work from `@wordpress/editor` (with maybe some warnings) but it should stay unchanged.

Reply:
Thanks for the heads up! Should I change it back to @wordpress/block-editor too?

Reply:
Yes.